cpg_workflows/stages/gatk_sv/gatk_sv_common.py

```diff
@@ -288,8 +288,8 @@ def make_combined_ped(cohort: Cohort | MultiCohort, prefix: Path) -> Path:
 
 
 def queue_annotate_sv_jobs(
-    cohort: Cohort | MultiCohort,
-    cohort_prefix: Path,
+    multicohort: MultiCohort,
+    prefix: Path,
```

Comment:
This is now always a multicohort, the cohort argument type/name is a legacy issue. Makes sense to fix it here